packages/terra-toolkit-docs/src/terra-dev-site/tool/webpackConfigTerra/upgrade-guide.6/Version 3.x.tool.mdx

```diff
@@ -0,0 +1,24 @@
+import { Badge } from '@cerner/webpack-config-terra/package.json?dev-site-package';
+
+<Badge />
+
+# webpack-config-terra Upgrade Guide
+
+## Changes from @cerner/webpack-config-terra 2.0.0 to @cerner/webpack-config-terra 3.0.0
+
+#### Updated Webpack-Dev-Server from version 3 to verison 4
+
+Webpack-dev-server has been updated to latest version 4.x. This update will fix hot reloading issue on development mode which was stopped working after upgrading Webpack to verison 5
+
+#### Breaking Changes
+Webpack-dev-server version 3 will not work with webpack-config-terra version 3.0.0. the webpack-dev-server has replaced the configuration options ( listed below ) in it's latest version v4, this changes will cause webpack-dev-server v3 configurations to fail with webpack-config-terra v3.
+- `inline` has been removed without replacement in webpack-dev-server v4.
+- `stats`, `index` and `publicPath` has been moved to `devMiddleware` option.
+
+Updating webpack-config-terra to version 3.0.0 would require consuming projects to update webpack-dev-server to version 4 in their local dependencies.
```

Comment:
We should avoid saying `3.0.0` when something is applicable to all of 3.x. Otherwise it looks like we're talking about `3.0.0` and not `3.0.1` or `3.1.0`. 

```suggestion
Updating webpack-config-terra to v3 would require consuming projects to update webpack-dev-server to version 4 in their local dependencies.
```

Reply:
updated : [2c8b7bb](https://github.com/cerner/terra-toolkit/pull/742/commits/2c845bf7ed693bc89065fc790213087e2fc27230#diff-2c8b7bbee9e59f9154faa0b2fb0859fe2ed8311baaea2639a6db1dfc7b8e75bf)